### PR TITLE
Added two missing functions in tool lib

### PR DESCRIPTION
**Pull request by rowaraujo**

### CODE REVIEW

File: data/actions/lib/actions.lua [hunk 1]
```diff
@@ -70,6 +70,27 @@ local function revertItem(position, itemId, transformId)
...
+		local thing = tile:getItemById(7186)
+		if thing then
+			thing:transform(7185)
+		end
+	end
+end
+
+local function revertHole(toPosition)
+	local tile = toPosition:getTile()
+
```
Comment (WibbenZ) [inline data/actions/lib/actions.lua]: Remove the empty line

Comment (rowaraujo) [inline data/actions/lib/actions.lua]: OK. Already removed.

**Comment by ninjalulz** [comment]
Remove the functions you've added, and replace revertIce/revertHole with revertItem. Don't forget to squash the commits.

```Lua
addEvent(revertIce, 60 * 1000, toPosition, 7186, 7185)
addEvent(revertHole, 30 * 1000, toPosition, 469, 8579)
```

**Comment by rowaraujo** [comment]
@ninjalulz Lol, Actions.lua Line 341 and Line 230 still uses the two functions.
You will have to make changes replacing them.

**Pull request closed without merging**